cmd/root.go

```diff
@@ -48,20 +48,19 @@ var rootCmd = &cobra.Command{
 		env.InitLogger()
 
 		source := env.Config().GetString("Source")
-		outputDir := env.Config().GetString("OutputDir")
 
 		configs := []ocp.Translator{}
-		ocpMaster := new(ocp.ConfigMaster)
-		ocpNode := new(ocp.ConfigNode)
-		ocpMaster.Add(source)
-		ocpNode.Add(source)
-		configs = append(configs, ocpMaster, ocpNode)
+		ocpOAuth := new(ocp.OAuthConfig)
```

Comment:
Just following up, this syntax isn't typically used in favor of literals, i.e. ocp.ConfigMaster{}. It's irrelevant here, but just passing that along.

Reply:
I wanted to change that one and now I know why, thanks for the clarification.

This could go in the next PR.